### PR TITLE
Add Avoid game

### PR DESCRIPTION
**Pull request by SongJungHyun1004**
Avoid is a simple game that a person avoid things from four directions.

It would be appreciated if you could give me some feedback.

### CODE REVIEW

File: freegames/avoid.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+from random import *
+from turtle import *
+import time
+from freegames import vector
+
+person = vector(0, 0)
+bombs1 = []
+bombs2 = []
+bombs3 = []
+bombs4 = []
```
Comment (grantjenks) [inline freegames/avoid.py:19]: Why track all four? Better to condense to a single list.

File: freegames/avoid.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    """Update object positions."""
+    person.move(aim)
+
+    for bomb1, bomb2, bomb3, bomb4 in zip(bombs1, bombs2, bombs3, bombs4):
+        bomb1.y -= 7
+        bomb2.y += 7
+        bomb3.x -= 7
+        bomb4.x += 7
+
+    if randrange(5) == 0:
```
Comment (grantjenks) [inline freegames/avoid.py:63]: Too many bombs by default!

File: freegames/avoid.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        bombs3.pop(0)
+    while len(bombs4) > 0 and not inside(bombs4[0]):
+        bombs4.pop(0)
+
+    if not inside(person):
+        draw(False)
+        return
+
+    for bomb1, bomb2, bomb3, bomb4 in zip(bombs1, bombs2, bombs3, bombs4):
+        if abs(bomb1 - person) < 15 or abs(bomb2 - person) < 15 or abs(bomb3 - person) < 15 or abs(bomb4 - person) < 15 :
```
Comment (grantjenks) [inline freegames/avoid.py:89]: I think the `freegames.vector` data type has an `overlap` method that you can use instead.

**Review by grantjenks — CHANGES_REQUESTED** [review]
It's an interesting game. But it's too hard currently. Also code is too redundant with `bombs[1-4]`. Simply to one `bombs` list instead.

**Comment by SongJungHyun1004** [comment]
First of all, I really appreciate for your feedback.

I corrected the redundant four list code.
But I don't know what to do about  freegames.vector  **overlap** method you said.
I couldn't find that method in https://grantjenks.com/docs/freegames/api.html#vectors

**Comment by SongJungHyun1004** [comment]
And I added **avoid.rst** file to docs.

**Pull request merged**